mantidproject/mantid
# Updated IMAT IDF for 2nd prototype detector

Description of work.

This is an attempt to have the most instrument definition files in the mantid/instrument/ folder.

As a secondary goal it should fix the detector ordering. The instrument view should now show a more correct version of IMAT than before, with each pixel being rendered.

Color fill plot after converting IMAT00003680 to dSpacing:(if it doesn't look like that then I failed copy pasting the correct data from the shared folder)
![image](https://cloud.githubusercontent.com/assets/9135965/23570282/6e7a1ebe-005b-11e7-8fdf-342b2023dfa8.png)

After diffraction focussing:
![image](https://cloud.githubusercontent.com/assets/9135965/23577081/63df79ec-00ae-11e7-9d14-08943c786355.png)

Other things to consider:
- There's a significantly large comment above the South Bank about how it's rotated, we should look into if that is desired for IMAT as well.
- Adding the diagnostics channels
- General cleaning up of the file.
^ All of those should be done in consecutive PRs (unless it's breaking the results). This PR is just to get a fast fix before Wednesday

**To test:**
1. Get IMAT00003680 
2. Run `ConvertUnits` to dSpacing
3. Do a color fill plot on the output dSpacing workspace. Look at the lines they should be straight
4. Run `DiffractionFocussing` on the dSpacing workspaces, the peaks should align.
4. Ask David to try doing the rest of the work now.
5. Profit 💰 

Fixes #18938 



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
